packages/components/src/components/Slider/__tests__/Slider.ct.tsx

```diff
@@ -0,0 +1,227 @@
+    let value = [20];
+    const component = await mount(
+        <Slider
+            aria-label="volume-control"
+            value={value}
+            onChange={(newValue) => {
+                value = newValue;
+            }}
+        />,
+    );
+    const slider = component.getByRole('slider');
+    const ariaValue = await slider.getAttribute('aria-valuenow');
+    expect(ariaValue).toBe('20');
+});
+
+test('should update values when mouse dragged', async ({ mount, page }) => {
+    const onChange = sinon.spy();
+    const onCommit = sinon.spy();
+
+    const component = await mount(
+        <Slider aria-label="volume-control" onChange={onChange} onCommit={onCommit} defaultValue={[50]} />,
+    );
+
+    const slider = component.getByRole('slider');
+
+    await dragSlider(page, component, slider, 10, onBeforeMouseUp);
+
+    function onBeforeMouseUp() {
+        expect(onCommit.notCalled).toBe(true);
+    }
+
+    sinon.assert.calledWithExactly(onChange, [60]);
+    sinon.assert.calledWithExactly(onCommit, [60]);
+    expect(await slider.getAttribute('aria-valuenow')).toBe('60');
+});
+
+test('should set and enforce min and max values', async ({ mount, page }) => {
+    const onChange = sinon.spy();
+    const onCommit = sinon.spy();
+    const wrapper = await mount(
+        <div style={{ position: 'relative', display: 'block', width: '300px', padding: '0 100px' }}>
+            <Slider
+                data-test-id={SLIDER_TEST_ID}
+                aria-label="range"
+                min={20}
+                max={40}
+                defaultValue={[30]}
+                onChange={onChange}
+                onCommit={onCommit}
+            />
+        </div>,
+    );
+    const component = wrapper.getByTestId(SLIDER_TEST_ID);
+
+    const slider = component.getByRole('slider');
+
+    expect(await slider.getAttribute('aria-valuemin')).toBe('20');
+    expect(await slider.getAttribute('aria-valuemax')).toBe('40');
+
+    await dragSlider(page, component, slider, -100);
+
+    sinon.assert.calledWithExactly(onChange, [20]);
+    sinon.assert.calledWithExactly(onCommit, [20]);
+    expect(await slider.getAttribute('aria-valuenow')).toBe('20');
+
+    await dragSlider(page, component, slider, 100);
+
+    sinon.assert.calledWithExactly(onChange, [40]);
+    sinon.assert.calledWithExactly(onCommit, [40]);
+    expect(await slider.getAttribute('aria-valuenow')).toBe('40');
+});
+
+test('should not interact with slider when disabled', async ({ mount }) => {
+    const component = await mount(<Slider aria-label="disabled-slider" disabled={true} defaultValue={[50]} />);
+
+    expect(await component.getAttribute('aria-disabled')).toBe('true');
+    expect(await component.getAttribute('data-disabled')).toBeDefined();
+
+    const pointerEvents = await component.evaluate((el) => getComputedStyle(el).pointerEvents);
+    expect(pointerEvents).toBe('none');
+});
+
+test('should enforce min steps between sliders', async ({ mount, page }) => {
```

Comment:
You are calling it thumb in the props
```suggestion
test('should enforce min steps between thumbs', async ({ mount, page }) => {
```